ssh/controlmaster.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2013 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ssh
+
+import (
+	"encoding/binary"
+	"errors"
+	"io"
+	"net"
+)
+
```

Comment:
Probably worth linking to whatever reference you're using; if you have a constant block (as suggested below) that makes a good place.

```
// This file implements the OpenSSH ControlMaster protocol, as described in
//   https://cvsweb.openbsd.org/src/usr.bin/ssh/PROTOCOL.mux?annotate=HEAD

// Protocol constants
const (
  controlMasterHello      = 0x00000001
  controlMasterAliveCheck = 0x10000004
)
```